samples/cellular/modem_shell/src/ping/icmp_ping.c

```diff
@@ -613,10 +648,10 @@ void icmp_ping_cmd_defaults_set(struct icmp_ping_shell_cmd_argv *ping_args)
 	ping_args->cid = MOSH_ARG_NOT_SET;
 	ping_args->pdn_id_for_cid = MOSH_ARG_NOT_SET;
 }
-
 /*****************************************************************************/
 
-int icmp_ping_start(struct icmp_ping_shell_cmd_argv *ping_args)
+static bool icmp_ping_current_conn_info_set(
+	struct icmp_ping_shell_cmd_argv *ping_args, struct icmp_ping_shell_cmd_argv *ping_argv)
```

Comment:
The function parameters could use some clarification. It's not clear what `ping_args` and `ping_argv` are for, especially when the function is called like this above:

`icmp_ping_current_conn_info_set(&used_ping_args, &used_ping_args)`

Reply:
OK, tried to clarify.